### PR TITLE
Add commands to deal with app settings

### PR DESCRIPTION
**Pull request by rdumont**
#### What is the purpose of this pull request?
It creates one new namespace (`settings`) with three new commands to deal with app settings, as follows.

```
settings <app> [field]                     Get app settings
settings set <app> <field> <value>         Set a value
settings unset <app> <field>               Unset a value
```

#### What problem is this solving?
This enables developers to manage apps' settings without leaving the command line.

#### How should this be manually tested?
1. Get an app's settings.
2 . Set a value and get settings again. The new value should be listed.
3. Get only a field from the app's settings.
4. Unset the field, then get settings again. The field should be gone.

#### Screenshots or example usage
settings-usage

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: src/modules/apps.js [hunk 1]
```diff
@@ -281,4 +282,57 @@ export default {
...
+    },
+
+    set: {
+      description: 'Set a value',
+      requiredArgs: ['app', 'field', 'value'],
+      handler: async (app, field, value) => {
+        try {
+          let jsonPath = '$.' + field
+          let patch = {}
+          jp.value(patch, jsonPath, value)
```
Comment (tamorim) [inline src/modules/apps.js]: @rdumont `path` cant receive the value from `jp.value`? This kind of mutation is quite awful...

Comment (tamorim) [inline src/modules/apps.js]: @rdumont Also, why wrap the whole code inside a try/catch? I think it would be better to wrap only where its needed.

**Comment by tamorim** [comment]
@rdumont There's a lot of variable declarations using `let` that could be using `const` instead.

File: src/modules/apps.js (hunk 1 above)
Comment (tamorim) [inline src/modules/apps.js]: @rdumont Don't forget you can use string interpolation using backticks and ${} inside.

Comment (rdumont) [inline src/modules/apps.js]: @tamorim, about the three issues:
1. No, the library doesn't support that :(
2. Sure, I was using that a lot to test, but they can be removed.
3. What line are you talking about?

Comment (tamorim) [inline src/modules/apps.js]: @rdumont on lines 296, 309 and 326.
But maybe you used the `+` operator for readability?

Comment (rdumont) [inline src/modules/apps.js]: Yes, I used it on purpose, since the string is a simple concatenation and contains a `$`. I find it more readable.

**Comment by rdumont** [comment]
@tamorim, I fixed the issues you mentioned.

**Comment by tamorim** [comment]
@rdumont LGTM :+1:

**Pull request merged**